src/sql/migrations/index.ts

```diff
@@ -0,0 +1,54 @@
+import { exit } from 'process'
```

Comment:
Why this file is in the same folder as migrations *.sql files?
The folder with migrations *.sql files should have no other files.